src/ape_ethereum/_print.py

```diff
@@ -35,8 +35,7 @@
 CONSOLE_CONTRACT_ID = cast(ChecksumAddress, "0x000000000000000000636F6e736F6c652e6c6f67")
 VYPER_PRINT_METHOD_ID = "0x23cdd8e8"  # log(string,bytes)
 
-console_contract = ContractType(abi=CONSOLE_LOG_ABI)
-console_contract.name = "console"  # TODO: Pretty confused why this can't be set in the constructor
+console_contract = ContractType(abi=CONSOLE_LOG_ABI, contractName="console")
```

Comment:
question: `name` versus `contractName`?

Reply:
`name` is an alias on our model for the `contractName` field in the spec. When making models, it expects the real field names and not the aliases.